data/config/datasets/adaguc.tests.datapostproc.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration>
+
+    <Layer type="database" hidden="true">
+        <Name>layer_windspeed_at_2000</Name>
+        <Title>Wind speed at 2000 meter</Title>
+        <FilePath filter=".*\.nc$" gfi_openall="true">{ADAGUC_PATH}/data/datasets/netcdf_5dims/netcdf_5dims_seq1/nc_5D_20170101000000-20170101001000.nc</FilePath>
+        <Variable orgname="data" long_name="Wind speed at 2000 meter">var_wind_speed_at_2000</Variable>
+        <Styles>auto</Styles>
+        <Dimension name="time">time</Dimension>
+        <Dimension name="member">member</Dimension>
+        <Dimension name="height" fixvalue="2000" hidden="true">elevation</Dimension>
+    </Layer>
+
+
+    <Layer type="database">
+        <Name>output</Name>
+        <Title>Wind speed at 2000 minus wind speed at 8000</Title>
+        <FilePath filter=".*\.nc$" gfi_openall="true">{ADAGUC_PATH}/data/datasets/netcdf_5dims/netcdf_5dims_seq1/nc_5D_20170101000000-20170101001000.nc</FilePath>
+        <Variable orgname="data" long_name="Wind speed at 8000 meter">var_wind_speed_at_8000</Variable>
+        <Dimension name="time">time</Dimension>
+        <Dimension name="member">member</Dimension>
+        <Dimension name="height" fixvalue="8000" hidden="true">elevation</Dimension>
+
+        <DataPostProc algorithm="include_layer" name="layer_windspeed_at_2000" mode="append"/>
+        <DataPostProc algorithm="operator" mode="-" a="var_wind_speed_at_2000" b="var_wind_speed_at_8000" name="output" units="m/s"/>
+        <Min>-1</Min>
+        <Max>1</Max>
+        <Styles>auto</Styles>
+
+    </Layer>
```

Comment:
Duplicate with `[data/config/datasets/adaguc.tests.datapostproc-windshear.xml](https://github.com/KNMI/adaguc-server/pull/271/files#diff-44afe170ffcc4675baeb5eb9d39079c36c85fe85e11c4259c95c6e7d708cd3dc)`?

Reply:
Renamed the contents so it matches with what it represents.